.drone-pandaria.yml

```diff
@@ -26,6 +26,9 @@ steps:
     settings:
       custom_dns: 1.1.1.1
     when:
+      ref:
+        include:
+          - "refs/heads/release/pandaria"
```

Comment:
`release/pandaria` or `pandaria` branch?

Reply:
Already changed to pandaria branch.